### DISCUSSION TITLE
Firebase functions config to set database url?

### DISCUSSION THREAD

**Discussion by JohnMarsden24** [prisma/prisma#11328] [Q&A] [answered]
Really want to integrate Prisma into our stack which consists of Firebase functions in JS + Node, however Firebase doesn't use `.env` files, it uses its own config file created via the cli. To use this config in a file you'd do this:

```javascript
const functions = require("firebase-functions");
const someKey = functions.config().someKey;
```

This doesn't work in the `schema.prisma` file so was wondering how I could get this to work? I was looking at [this answer](https://github.com/prisma/prisma/issues/2443#issuecomment-630679118) of overriding the database url as I have access to my config then, however I wouldn't know what to set the initial database url as in my `schema.prisma` file. Any ideas?

**Comment by janpio** [comment, marked as answer]
Here is an example from our e2e test suite: https://github.com/prisma/e2e-tests/tree/dev/platforms-serverless/firebase-functions

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
